### DISCUSSION TITLE
Change legend font color

### DISCUSSION THREAD

**Discussion by plazasnico** [sgratzl/chartjs-chart-geo#79] [Q&A]
How is possible to change the font color of legend scales?  
Since I need the white font color because my background color is somewhat darkness and if would be possible changing the font style such as font size and font weight too.

Now I'm using the 2.1.0 version because my react project implements chart.js 2.x version and might have some issues when migrating to some recent version.

![image](https://user-images.githubusercontent.com/81947183/123216964-b199f180-d48f-11eb-8f3e-59aa2b3712fd.png)

Thanks!

**Comment by sgratzl** [comment]
legends are just normal scales/axis, thus take a look at the chart.js documentation for changing scale/axis colors and fonts.

**Comment by plazasnico** [comment]
¡Great thanks!
